scripts/tests/api_compare/docker-compose.yml

```diff
@@ -2,70 +2,185 @@
 version: "3.8"
 
 services:
-  # Initialisation service. Downloads the shared assets to facilitate the setup of the other services.
-  init:
-    image: ${FOREST_IMAGE}
-    user: root
+  forest:
+    build:
+      context: ../../../.
+      dockerfile: ${FOREST_DOCKERFILE_OVERRIDE:-Dockerfile}
+    volumes:
+      - node-data:/data
+      - filecoin-proofs:${FIL_PROOFS_PARAMETER_CACHE}
+    networks:
+      - api-tests
+    environment:
+      - FIL_PROOFS_PARAMETER_CACHE=${FIL_PROOFS_PARAMETER_CACHE}
+      - FULLNODE_API_INFO=/dns/forest/tcp/${FOREST_RPC_PORT}/http
+    entrypoint: [ "/bin/bash", "-c" ]
+    command:
+      - |
+        set -euxo pipefail
+        # Setup data_dir to avoid downloading snapshot on every local run
+        echo "[client]" > config.toml
+        echo "data_dir = \"/data/forest\"" >> config.toml
+        forest --config config.toml --chain ${CHAIN} --encrypt-keystore false --no-gc --height=-900 --auto-download-snapshot --rpc-address 0.0.0.0:${FOREST_RPC_PORT}
+    healthcheck:
+      test: [ "CMD", "forest-cli", "chain", "head" ]
+      interval: 15s
+      timeout: 3s
+      retries: 3
```

Comment:
Would it make sense to put `start_period`?

Reply:
`forest` starts RPC server before importing a snapshot so it's not necessary, I can give it maybe 30s?

Reply:
Fixed.